src/skin/popup.html

```diff
@@ -90,14 +90,17 @@ <h1 id="share_title" class="i18n_share_title overlay_title"></h1>
   </div>
 
 <div id='privacyBadgerHeader'>
-  <a id="options" title="i18n_popup_options_button" href='/skin/options.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img src="/icons/options.svg" alt="i18n_popup_options_button"></a>
-  <a id="share" title="i18n_popup_share_button" class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" height="auto" src="/icons/share.svg" alt="i18n_popup_share_button"></a>
-  <a id="help" title="i18n_popup_help_button" href='/skin/firstRun.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img src="/icons/help.svg" alt="i18n_popup_help_button"></a>
+  <a id="options" title="i18n_popup_options_button" href='/skin/options.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" src="/icons/black-options-icon.svg" alt="i18n_popup_options_button"></a>
```

Comment:
Why don't we just replace the existing icon files with the new icon files? No need for new filenames, is there?